docs/content/flow-port/faq.md

```diff
@@ -149,3 +149,7 @@ Please see a list [here](https://github.com/onflow/flow/blob/master/nodeoperator
        - a.) No you do not! You only need your Ledger device to sign transactions. If you simply want to view your account, you can do so without your Ledger. You can do this by navigating directly to the appropriate desired page URL, while inputting your address into the URL itself. For easy reference, below is a list of these URLs and where you would input your address:
        - Dashboard: https://port.onflow.org/account/[AccountAddress]
        - Stake & Delegate: https://port.onflow.org/stake-delegate/[AccountAddress]
+
+  7. I am clicking 'submit' to execute a transaction, but nothing is happening. How can I unblock myself?
+  
+       - a.) Please disable any pop-up blockers you have and refresh the page. If you are still experiencing issues, please reach out via our [discord](https://discord.gg/4yGnMzkZxr) in the appropriate channel.
```

Comment:
```suggestion
       - a.) Please disable any pop-up blockers you have and refresh the page. If you are still experiencing issues, please reach out via [Discord](https://discord.gg/4yGnMzkZxr) in the appropriate channel.
```

Reply:
Maybe we should add a Port channel to Discord too?

Reply:
I already brought that up to wei but he said no for now.